src/terraform/structure/terraform_module_test.go

```diff
@@ -16,11 +16,6 @@ func TestTerrraformModule(t *testing.T) {
 		assert.True(t, isRemote)
 	})
 
-	t.Run("Test TF Module remote scalr logic", func(t *testing.T) {
-		isRemote := isRemoteModule("jameswoolfenden.scalr.io/acc-u1ksa0vgdflusgo/cloudfront/aws")
-		assert.True(t, isRemote)
-	})
```

Comment:
@mhennecke Why is this test not needed anymore? AFAIK, this should still work, right?

Reply:
@nimrodkor It is still there. I just moved it a bit further down in the file.